commons-eid-bom/pom.xml

```diff
@@ -1,12 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
+	<parent>
```

Comment:
Reference to the parent POM within the BOM was not provided to keep the dependency management clean within the project where you import the BOM.
Also, AFAICR referring to the parent POM within the BOM made the maven-release-plugin to freak out.